extensions/ql-vscode/src/model-editor/method-modeling/method-modeling-view-provider.ts

```diff
@@ -39,6 +42,17 @@ export class MethodModelingViewProvider implements WebviewViewProvider {
     webviewView.webview.html = html;
 
     webviewView.webview.onDidReceiveMessage(async (msg) => this.onMessage(msg));
+
+    this.webviewView = webviewView;
+  }
+
+  public async setMethod(method: ExternalApiUsage): Promise<void> {
+    if (this.webviewView) {
```

Comment:
This doesn't seem to be working for me. At the point it reaches this point `this.webviewView` is undefined. Does it work for you? 🤔